prisma/schema.prisma

```diff
@@ -10,19 +10,71 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum StationType {
+  Fixed
+  Mobile
+}
+
 model Station {
-  id   String @id @default(auto()) @map("_id") @db.ObjectId
-  name String
-  type String
-  sensors Sensor[]
-  deletedAt DateTime?
+  id            String   @id @default(auto()) @map("_id") @db.ObjectId
+  name          String
+  type          StationType
+  latitude      String
+  longitude     String
+  description   String
+  image_url     String
+  sensors       Sensor[]   // Define a relation to Sensor
+}
+
+enum SensorType {
+  LittObs
+  SensOcean
 }
 
 model Sensor {
-  id   String @id @default(auto()) @map("_id") @db.ObjectId
-  identifier String @unique
-  stationId String? @db.ObjectId
-  station Station? @relation(fields: [stationId], references: [id])
+  id            String   @id @default(auto()) @map("_id") @db.ObjectId
+  identifier    String   @unique
+  type          SensorType
+  nbr_measures  Int
+  stationId     String?  @db.ObjectId // Add a reference to the Station model
+  station       Station? @relation(fields: [stationId], references: [id])
+  records       Record[]    // Add a one-to-many relationship to Record
+  files         File[]      // Add a one-to-many relationship to File
```

Comment:
We should add this properties to track the changes by user : 
- created_at
- created_by 
- updated_at
- updated_by